### ISSUE TITLE
Rv0678 and mmpR5 on report file

### ISSUE THREAD

**Issue by cimendes** [jodyphelan/TBProfiler#292]
Hi @jodyphelan!

I've been using the JSON output and parsing it out for downstream validation of the results and I came across something that I found a bit odd. 

I'm under the impression that mmpR5 is always referred to as Rv0678 but I came across this example where mmpR5 is referred to instead. 
[Dassie_bacillus02.results.zip](https://github.com/jodyphelan/TBProfiler/files/12029647/Dassie_bacillus02.results.zip)

Am I under the wrong impression?

Thank you!

**Comment by jodyphelan** [comment]
You are not wrong. Rv0678 is the gene id and [mmpR5](https://www.uniprot.org/uniprotkb/I6Y8F7/entry) is the gene name. I think when the bedaquiline resistance mechanisms was first discovered this gene hadn't been annotated properly yet so it only had a gene id (Rv0678). Since then the name kind of stuck and most papers still refer to it by the id. Both are presented in the output:
```
      ...
      "locus_tag": "Rv0678",
      "gene": "mmpR5",
      ...
```
